circle-mlir/circle-mlir/tools-test/gen-onnx/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+# python3 venv folder
+# NOTE Docker image for CI doesn't use venv
+set(VENV_PATH "${CMAKE_SOURCE_DIR}/infra/overlay/venv")
+
+# Copy test scripts
+unset(GEN_SCRIPT_DEPS )
+macro(COPY_SCRIPT FILENAME)
+  set(SCRIPT_SRC "${CMAKE_CURRENT_SOURCE_DIR}/${FILENAME}")
+  set(SCRIPT_DST "${CMAKE_CURRENT_BINARY_DIR}/${FILENAME}")
+  add_custom_command(
+    OUTPUT ${SCRIPT_DST}
+    COMMAND ${CMAKE_COMMAND} -E copy "${SCRIPT_SRC}" "${SCRIPT_DST}"
+    DEPENDS ${SCRIPT_SRC}
+    COMMENT "gen-onnx: prepare ${FILENAME}"
+  )
+  list(APPEND GEN_SCRIPT_DEPS "${SCRIPT_DST}")
+endmacro(COPY_SCRIPT)
+
+COPY_SCRIPT(run_gen_onnx.sh)
+COPY_SCRIPT(run_gen_onnx.py)
+
+# Models folder for unit testing
+set(PYTORCH_UNIT_PATH "${CMAKE_SOURCE_DIR}/models/unit")
+set(PYTORCH_NET_PATH "${CMAKE_SOURCE_DIR}/models/net")
```

Comment:
Is `models/net` going to be introduced later? 

I'd like to understand the difference between `units` and `net`. How do we determine which module belongs to `units` or `net`?

Reply:
> Is models/net going to be introduced later?

Yes

> difference between units and net

`unit` is for single Op, `net` is for multiple Ops; a network for Ops.